calcom/cal.com
# Grab currency from payment credentials

## What does this PR do?



This PR grabs a user's currency from their payment app credentials and renders it properly on the availability and booking page.

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Add Stripe to your account
- Edit an event type to include a price
- When going to the availability and booking page of that event type the currency should render properly

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
